src/content/reference/platform/language.md

```diff
@@ -0,0 +1,54 @@
+
+# Languages
+
+## Background
+
+Workers is a polyglot platform. You can likely write code on Workers with a language you already know. We originally built Workers to use JS/TS and over time we've added support for new languages. We plan to add popular languages like Python and continue to deepen support for added languages.
```

Comment:
"plan to add popular languages like python" is in contrast to the serverless week blog posts "excited to announce support for Python"

I wouldn't say we originally built workers to use JS/TS, I'd say we built it to use JavaScript and WebAssembly